code/modules/reagents/reagents/drugs.dm

```diff
@@ -273,3 +273,83 @@
 	M.stats.addTempStat(STAT_TGH, -STAT_LEVEL_BASIC, STIM_TIME, "sanguinum_w")
 	M.stats.addTempStat(STAT_COG, -STAT_LEVEL_BASIC, STIM_TIME, "sanguinum_w")
 	M.stats.addTempStat(STAT_ROB, -STAT_LEVEL_BASIC, STIM_TIME, "sanguinum_w")
+
+/datum/reagent/drug/roachbeer
+	name = "Kakerlakenbier"
+	id = "roachbeer"
+	description = "A green-ish substance made out of diplopterum, beer and fuel mixed with water. Doesn\'t look nor smell like beer..."
+	taste_description = "heady roach guts"
+	reagent_state = LIQUID
+	color = "#123a15"
+	metabolism = REM
+	nerve_system_accumulations = 40
+	addiction_chance = 60
+	overdose = REAGENTS_OVERDOSE
+	addiction_threshold = 15
+	sanity_gain_ingest = 2
+	taste_tag = list(TASTE_SOUR, TASTE_BUBBLY)
+
+/datum/reagent/drug/roachbeer/affect_ingest(mob/living/carbon/M, alien, effect_multiplier) ////// checks user for having a vagabond perk,
+	var/perk_check = effect_multiplier
+	if(M.stats.getPerk(PERK_VAGABOND))														////// increases sanity_gain to 4 if true
+		perk_check = effect_multiplier * 2
+	apply_sanity_effect(M, perk_check)
+	M.slurring = max(M.slurring, 30)
```

Comment:
Can't you just... Make it you actually drunk instead of slurring and lower the sanity bonus somewhat because you're drunk?

Reply:
no, I can't give a drug ethanol-like strength because coder spaghet, but I will tweak everything else now that the methe pr got closed.